sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePage.java

```diff
@@ -6,26 +6,25 @@
 import com.azure.core.util.IterableStream;
 
 /**
- * Represents Page from service that has reference to next set of one or more pages,
- * such a reference is known as continuation token.
- *
- * @param <C> Type of the continuation token
- * @param <T> Type of the elements in the page
+ * Represents a page returned from a service, this page may contain a reference to additional pages known as a
```

Comment:
nit: A page may not necessarily have to be returned from a service.

Reply:
Removed service reference